mind_matter_api/app.py

```
@@ -32,10 +33,16 @@ def create_app(config_object="mind_matter_api.settings"):
     :param config_object: The configuration object to use.
     """
     app = Flask(__name__, root_path=os.path.dirname(os.path.abspath(__file__)))
-    CORS(app, resources={r"/*": {"origins": "*"}}, supports_credentials=True)  # <--- THIS LINE
-
+    CORS(app, resources={r"/*": {"origins": "*"}}, supports_credentials=True)  # <
+    
+    # Rate limiter using flask-caching (memcached )
+    
     logger = logging.getLogger(__name__)
```

Ruff (0.8.2)

Local variable `logger` is assigned to but never used

Remove assignment to unused variable `logger`

(F841)